code/modules/paperwork/photocopier.dm

```diff
@@ -115,35 +175,38 @@
 	if(.)
 		return
 
+	if(machine_stat & (BROKEN|NOPOWER))
+		return
+
 	switch(action)
 		// Copying paper, photos, documents and asses.
 		if("make_copy")
 			if(check_busy(usr))
 				return FALSE
 			// ASS COPY. By Miauw
 			if(ass)
-				do_copy_loop(CALLBACK(src, PROC_REF(make_ass_copy), usr), usr)
+				if(ishuman(ass) && (ass.get_item_by_slot(ITEM_SLOT_ICLOTHING) || ass.get_item_by_slot(ITEM_SLOT_OCLOTHING)))
+					to_chat(usr, span_notice("You feel kind of silly, copying [ass == usr ? "your" : ass][ass == usr ? "" : "\'s"] ass with [ass == usr ? "your" : "[ass.p_their()]"] clothes on.") )
+					return FALSE
```

Comment:
```suggestion
					if(ass == usr)
						to_chat(usr, span_notice("You feel kind of silly, copying your ass with your clothes on."))
					else
						to_chat(usr, span_notice("You feel kind of silly, copying [ass]\'s ass with [ass.p_their()] clothes on."))
					return FALSE
```

we don't need to check the same check three separate times. i'm not a fan of the indentation but i dislike how overly complicated/wasteful the string is